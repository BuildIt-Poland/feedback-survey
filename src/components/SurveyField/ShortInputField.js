import React from 'react';
import { FormField } from 'grommet';

import { fontWeightNormal, colorWhite } from '../../styles/variables';
import { formField } from '../../types';

export const inputTheme = {
  global: {
    input: {
      weight: fontWeightNormal
    },
    focus: {
      border: {
        color: colorWhite
      }
    }
  }
};

const ShortInputField = ({ field }) => {
  return (
    <FormField name={field.id} label={field.content} required={field.required} placeholder="Type your answer here" />
  );
};

ShortInputField.propTypes = {
  field: formField.isRequired
};

export default ShortInputField;
