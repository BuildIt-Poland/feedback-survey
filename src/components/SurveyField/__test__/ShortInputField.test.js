import React from 'react';
import { create } from 'react-test-renderer';

import ShortInputField from '../ShortInputField';

describe('COMPONENT - SurveyField ShortInputField', () => {
  it('renders correctly', () => {
    const component = create(<ShortInputField />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
