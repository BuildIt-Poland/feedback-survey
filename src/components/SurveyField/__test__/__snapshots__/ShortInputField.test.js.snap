// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - SurveyField ShortInputField renders correctly 1`] = `null`;
